docker/build.sh

```diff
@@ -0,0 +1,19 @@
+set -x
+set -e
+TARBALL=$(nix build\
+	--no-link\
+	--print-out-paths\
+	--accept-flake-config\
+	--accept-flake-config\
```

Comment:
duplicate line?

Reply:
mmyes, good catch. will fix with the rest of your inevitable comments